Utils/test_upload_flow/verify_bucket.py

```diff
@@ -13,25 +13,32 @@
 from Tests.scripts.utils.log_util import install_logging
 from Tests.scripts.utils import logging_wrapper as logging
 
+# Info log Messages
+VERIFY_NEW_PACK = "verified the new pack {} in the index and that version {} zip exists under the pack path {}"
+VERIFY_MODIFIED_PACK = "verified the packs new version is in the index and that all the new items are present in the pack"
+VERIFY_NEW_VERSION = ("verified the new pack's version exists in the index and that the release notes is parsed correctly "
+                      "in the changelog")
+VERIFY_RN = "verified the content of the release notes is in the changelog under the right version"
+VERIFY_HIDDEN = "verified the pack does not exist in index"
+VERIFY_README = "verified the readme content is parsed correctly and that there was no version bump if only readme was modified"
+VERIFY_FAILED_PACK = "verified the commit hash is not updated in the pack metadata in the index.zip"
+VERIFY_MODIFIED_ITEM_PATH = "verified the path of the pack item is modified"
+VERIFY_DEPENDENCY = "verified the new dependency is in the pack metadata"
+VERIFY_NEW_IMAGE = "verified the new image was uploaded"
+VERIFY_HIDDEN_DEPENDENCY = "verified the hidden dependency pack not in metadata.json"
+VERIFY_PACK_NOT_IN_MARKETPLACE = ("verified the pack {} is NOT in the index and that version 1.0.0 zip DOES NOT "
+                                  "exists under the pack path in {} bucket")
```

Comment:
Let's open an Enum Class for the messages
Call it `VerifyMessages` and remove the `VERIFY_` from the variable names